docs/usage.md

```diff
@@ -8,7 +8,23 @@ The software has been tested with MATLAB R2021B on an Ubuntu 20.04 LTS OS. The r
 - MATLAB Curve Fitting Toolbox
 
 ### Easier way to install dependencies
-To be updated.
+Instead of manually installing the iDynTree MATLAB bindings, we provide a script that installs the bindings using a `Mambaforge` installer without modifying anything in the PC.
+
+One can install these bindings by launching the script `install_idyntree_bindings` on the MATLAB console while being in the `scripts` folder. This process must install the bindings in `paper_ramadoss-2022-ral-humanoid-base-estimation/deps/idyntree-matlab/mex`.
```

Comment:
```suggestion
One can install these bindings by launching the script `install_idyntree_bindings` on the MATLAB console while being in the `scripts` folder. This process installs the bindings in `paper_ramadoss-2022-ral-humanoid-base-estimation/deps/idyntree-matlab/mex` directory.
```

Reply:
Fixed in https://github.com/ami-iit/paper_ramadoss-2022-ral-humanoid-base-estimation/commit/2295787f336f3825bc6366453dbd792b52198801